daiquiri/query/static/query/js/services/plot.js

```diff
@@ -1,208 +1,569 @@
+
+  /* get the base url */
+  var baseurl = angular.element('meta[name="baseurl"]').attr('content');
+
+  /* configure resources */
+  var resources = {
+    rows: $resource(baseurl + 'serve/api/rows/')
+  }
+
+  var validtypes = ['short', 'int', 'long', 'float', 'double', 'boolean'];
+  var excluded_ucd = ['meta.id', 'meta.version'];
```

Comment:
I couldn't think of any use in terms of plotting, so I excluded it. Any idea why people would like to plot against `meta.id`?

Reply:
Maybe I misunderstood, but I think all meta.id columns will be not plottable, corect? How ever the unique source identifier from a catalogue should be plottable at least in the x axis. But maybe there is a way to only keep the unique identifier. Otherwise there is also the datalink unique ID that maybe useful. In XMMSSC we could imagine a 2D scatter plot with color range showing ra and dec and colouring according to the stack/field/observation ID. stuff like that. This is not very scientific but for quick exploration/verification I think that is quite useful.